### ISSUE TITLE
Eslint complains about `react/jsx-fragments` rule syntax

### ISSUE THREAD

**Issue by gdepouilly** [eslint/eslint#12657]
**Tell us about your environment** 
```
Environment Info:
Node version: v12.6.0
npm version: v6.13.3
Local ESLint version: Not found
Global ESLint version: Not found

➜ npx eslint --version 
v6.7.2
```

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-ESLint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  parserOptions : {
    ecmaFeatures : {
      legacyDecorators : true
    }
  },
  extends : ["@xxx/xxx/react"], // custom rules package
  rules : {
    "import/no-unresolved" : "off",
    "no-magic-numbers"     : ["error", {
      ignore             : [-1, 1, 0, 2, 3, 100, 1000],
      ignoreArrayIndexes : true,
      enforceConst       : true,
      detectObjects      : false,
    }],

  "react/destructuring-assignment" : "off",
  "react/forbid-prop-types" : "off", 
  "react/jsx-props-no-spreading" : [2, {
    html : "ignore",
    exceptions : ["WrappedComponent"]
  }],
  "react/jsx-fragments" : "element"
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
npx eslint myfile.js 
```

**What did you expect to happen?**
No error is thrown.

**What actually happened? Please include the actual, raw output from ESLint.**
```
.eslintrc.js:
        Configuration for rule "react/jsx-fragments" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"element"').
```
ESlint throws an error about a `severity` setting which does not exists for this rule, see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by yeonjuan** [comment]
@gdepouilly 
It's because there is no specified rule severity. Maybe it is the `<enabled>` in the document you mentioned - [jsx-flagments](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md)). So try it

In your configuration, Change `"react/jsx-fragments" : "element"` to one of belows.
  * `"react/jsx-fragments" : ["error", "element"]`
  * `"react/jsx-fragments" : ["warn", "element"]`
  * `"react/jsx-fragments" : ["off", "element"]`

and you can find the documentation on how to configure rules [here](https://eslint.org/docs/user-guide/configuring#configuring-rules)

**Comment by gdepouilly** [comment]
Hi, thanks for your quick answer. Actually, it works with your example. 

Just to be curious, I upgraded from version `5.16.0`, which was working. Did the rule syntax changed since ? I didn't find any commit relative to my problem in the changelog :thinking:

**Comment by ljharb** [comment]
@gdepouilly what that means is that it wasn't actually working; eslint just wasn't complaining about your broken configuration. All eslint rules, since forever, probably for forever, require a severity.

**Comment by kaicataldo** [comment]
Closing since this question has been answered.
